awscli/examples/elasticbeanstalk/update-environment.rst

```diff
@@ -35,6 +35,9 @@ The following command sets the value of the "PARAM1" variable in the "my-env" en
 
 The ``option-settings`` parameter takes a namespace in addition to the name and value of the variable. Elastic Beanstalk supports several namespaces for options in addition to environment variables.
 
+**To set multiple environment variables in shorthand form**
+aws elasticbeanstalk update-environment --environment-name my-env --option-settings Namespace=aws:elasticbeanstalk:application:environment,OptionName=PARAM1,Value=ParamValue Namespace=aws:elasticbeanstalk:application:environment,OptionName=PARAM2,Value=Param2Value
```

Comment:
Would you be able to add a similar example to [create-environment.rst](https://github.com/aws/aws-cli/blob/develop/awscli/examples/elasticbeanstalk/create-environment.rst)? That should cover our bases with respect to setting multiple environment variables using the `--option-settings` parameter for the `elasticbeanstalk` command.